internal/receiver/smartagentreceiver/output.go

```diff
@@ -223,7 +232,28 @@ func (output *Output) SendEvent(event *event.Event) {
 }
 
 func (output *Output) SendSpans(spans ...*trace.Span) {
-	output.logger.Debug("SendSpans has been called.", zap.Any("Span", spans))
+	if output.nextTracesConsumer == nil {
+		return
+	}
+
+	for _, span := range spans {
+		for name, value := range output.defaultSpanTags {
+			if span.Tags == nil {
```

Comment:
Would it work if this block were in the outer loop?

Reply:
definitely, thanks for catching.